### PR TITLE
fix: TRLST-339 malicious file upload

### PR DESCRIPTION
**Pull request by davecharles**
- Updated deps to use new chunk uploader
- Bumped boto and django-storages to match
- Removed and ignored sqlite3 db that is built on deployment
- Update view to use new handler and detect malware response
- Added necessary settings to django config
- Clean-up, removed cruft routes

### CODE REVIEW

File: trade_remedies_public/cases/views.py [hunk 1]
```diff
@@ -1206,41 +1210,6 @@ def get(
             },
         )
 
-    def post(
```
Comment (rossmiller) [inline trade_remedies_public/cases/views.py:1209]: Why was this removed?

**Review by rossmiller — COMMENTED** [review]
One question.

File: trade_remedies_public/cases/views.py (hunk 1 above)
Comment (davecharles) [reply trade_remedies_public/cases/views.py:1209]: It had the second request.FILES access and it turned out there is no route to POST on that view - looks like at some point it was superseded by UploadDocumentsView

**Review by rossmiller — APPROVED** [review]
LGTM!
